strip-path/stripPath.js

```diff
@@ -1,6 +1,29 @@
 
-function stripPath(...args) {
-  return args;
+const normalizePath = (pathToNormalize) => {
+  const pathSegments = pathToNormalize.split('/');
+  const newPath = [];
+  for (const segment of pathSegments) {
+    if (segment === '..') {
+      newPath.pop();
+    } else {
+      newPath.push(segment);
+    }
+  }
+  return newPath.join('/');
+};
+
+function stripPath(pathToBeStriped, pathToStrip) {
```

Comment:
@rakeshtembhurne It would be better if you could give more descriptive names for the parameters of your function. First parameter in my opinion is pathToStrip and second parameter is partOfPathToBeStriped. That's more understandable I guess :)

Reply:
Good catch. IMHO we are not 100% sure that the second parameter will be a part of the given path to be stripped. Also, it's not clear whether the second parameter will be a part of or exactly the same as the first parameter. So I don't think 'partOf' is needed here.